docs/development/maintenance.md

```diff
@@ -13,1292 +15,1464 @@ modules of ILIAS.
+2. **Authority to Sign off Conceptual Changes**: The people listed here are
+authorised to contribute directly to the code base of the ILIAS core. They are
+authorised to commit directly to the codebase of the ILIAS core and to merge
+Pull Requests. They are the ones deciding on the structure and quality of the
+code of a component.
+3. **Authority to Curate Test Cases**: The people listed here are
+authorised to modify and delete existing test cases. They also have the final
+say on new test cases and can ask for modifications. They will be the ones
+contacted if there are questions concerning the test cases for a component.
+4. **Authority to (De-)Assign Authorities**: The people listed here are
+authorised to assign and deassign other people to the authorities of a component
+They are the only ones allowed to modify the `maintanance.json` of a component.
+
+Each of these authorities can be held by a different set of people. This means
+that the social organisation of different groups working on different parts of
+the code of ILIAS can be different.
+Right now ILIAS knows a few different social structures for the maintenance of
+units in the code of ILIAS:
+
+* In the **"Classic Model"** all authorities are concentrated in one person and
+this person works mostly alone.
+* In the **Coordinator Model** all authorities are concentrated in one or more
+people and they work together with other developers in the community to improve
+the code.
+* In the **"Test and Assessment Model"** the authorities **to Sign off Conceptual
+Changes**, **to Sign off Testcase Changes**, and **to (De-)Assign Authorities**
+lie with one person and the **Authority to Sign off Code Changes** with two
+others.
+
+More will surely emerge as the optimal solution for each unit is found.
+
+# Responsibilites
+Independently of the social organisation, for each [component](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/components-and-directories.md) the following
+responsibilites need to be assumed:
+
+* All people holding an authority must agree to coordinate the development
+of their [component](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/components-and-directories.md)
+with the Product Manager and with the people maintaining other units of code.
+* One of the people holding either the **Authority to Sign off Code Changes** or
+the **Authority to Sign off Conceptual Changes** gets assigned related bugs
+automatically by the [Issue-Tracker](https://mantis.ilias.de). S/he is responsible
+to make sure all issues receive a response within the defined time frame and are
+either fixed in a timely manner or postponed/closed with a solid explanation.
+* The people holding the **Authority to Sign off Code Changes** are responsible
+for pull requests to their component and get assigned related pull requests
+by the Technical Board according to the [Rules for Maintainers and Coordinators
```

Comment:
Linkname and title should be changed? Also, maybe the TB does not assign all PRs in the Future.